config/101-podsecuritypolicy.yaml

```diff
@@ -16,6 +16,11 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: tekton-triggers
+  labels:
+    app.kubernetes.io/name: psp
```

Comment:
Any reason for using `psp` instead of a "long" name ?

Reply:
Shall i add the `app.kubernetes.io/name` label at all ?

Reply:
I mean, we want to use label for two things:
- identify the application/component the resource belongs to
- be able to look up the resource when necessary

I don’t think we need to look up PSP anywhere and other labels are enough to identify the application so...

Reply:
yeah, I don't think `app.kubernetes.io/name` has too much value indeed.